acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/configuration/BesuNodeConfigurationBuilder.java

```diff
@@ -211,8 +211,8 @@ public BesuNodeConfigurationBuilder permissioningConfiguration(
     return this;
   }
 
-  public BesuNodeConfigurationBuilder keyFilePath(final String keyFilePath) {
-    this.keyFilePath = Optional.of(keyFilePath);
+  public BesuNodeConfigurationBuilder keyFilePath(final Optional<String> keyFilePath) {
```

Comment:
Why do we need to make this optional? Isn't the idea of the builder that if you don't need this value you just don't call the `keyFilePath ` method?

Reply:
And it looks like `keyFilePath` already starts as an empty.